### ISSUE TITLE
Weather macro disappears

### ISSUE THREAD

**Issue by Accelsnow** [willasm/vc-weather#4]
When I use in-note macro `%weather3%`, the macro simply disappears. It worked before on earlier versions.

Macro input:
![image](https://github.com/user-attachments/assets/3735b8fb-b5ea-48b8-abd1-a01daac75ba9)

When I hit %:
![image](https://github.com/user-attachments/assets/6fc37cf0-7421-4d24-896f-43c4f6b90eb4)

There is no error log in console. I verified that my API key works, as the toolbar and HTML weather works:
![image](https://github.com/user-attachments/assets/49665c26-f4f8-4034-9730-cacad6b6fbae)
![image](https://github.com/user-attachments/assets/ec21148c-4238-4274-b286-c74590635897)
![image](https://github.com/user-attachments/assets/2be5a06a-1c0f-49ed-91d9-ff10b336314d)
![image](https://github.com/user-attachments/assets/e01f75ab-32e4-42d5-a564-e8fd04383a39)

My weather3 template:

```
Brief
%dow1-today%🔸%address%🔸%conditions-today%🔸(%feelslikemin-today%, %feelslikemax-today%) [%tempmin-today%, %tempmax-today%]🔸%preciptype-today% %precipcover-today%
```

Obsidian v1.6.7
Plugin v0.1.3

**Comment by willasm** [comment]
Fixed in 0.1.4

Sorry about that, needed to make changes to the insertion code for submission to Obsidian and introduced the bug then. Please test this and let me know if you are getting the correct template inserted or not.

Thanks, William

**Comment by Accelsnow** [comment]
> Fixed in 0.1.4
> 
> Sorry about that, needed to make changes to the insertion code for submission to Obsidian and introduced the bug then. Please test this and let me know if you are getting the correct template inserted or not.
> 
> Thanks, William

The bug is fixed. Thanks William really appreciate the plugin :)
